apps/client/src/app/components/account-detail-dialog/account-detail-dialog.html

```diff
@@ -69,6 +69,14 @@
       [mat-stretch-tabs]="false"
       [ngClass]="{ 'd-none': isLoadingActivities }"
     >
+      <mat-tab>
+        <ng-template i18n mat-tab-label>Holdings</ng-template>
+        <gf-holdings-table
+          [baseCurrency]="user?.settings?.baseCurrency"
+          [locale]="user?.settings?.locale"
```

Comment:
Please add `[deviceType]="deviceType"`

Reply:
Sure, but do you mind telling me what this field is about though? It doesn't seem to be used and my IDE is even complaining about an "unresolved variable".

Reply:
Ok, found it. It's actually data.deviceType here, that's why it said "unresolved"

Reply:
> Sure, but do you mind telling me what this field is about though?

The visible columns are controlled via `deviceType`. On mobile, the holdings table is simplified.

> It's actually data.deviceType here

Oops! 🙈